packages/conventional-changelog-core/lib/merge-config.js

```diff
@@ -20,10 +20,21 @@ var _ = require('lodash');
 var rhosts = /github|bitbucket|gitlab/i;
 var rtag = /tag:\s*[v=]?(.+?)[,\)]/gi;
 
+function semverTagsPromise(options) {
+  var semverTagsDeferred = Q.defer();
```

Comment:
Can we use the `Promise` api instead of `defer()`? [Reason](https://github.com/petkaantonov/bluebird/wiki/Promise-anti-patterns#the-deferred-anti-pattern)